packages/backend/src/App.ts

```diff
@@ -286,6 +286,7 @@ export default class App {
             const instanceConfigurationService =
                 this.serviceRepository.getInstanceConfigurationService<InstanceConfigurationService>();
             await instanceConfigurationService.initializeInstance();
+            await instanceConfigurationService.updateInstanceConfiguration();
```

Comment:
If the server restarts, will `initializeInstance` still run? When would the server start and `updateInstance` runs but not `initializeInstance`?

Reply:
both will run, it is up to the methods to decide what to do, depending on the config, and if orgs/projects exist. 

In initializeInstance we will check if there are ogs already, if there is at least one, nothing will happen.